lib/chef/resource/windows_pagefile.rb

```diff
@@ -24,7 +24,7 @@ class WindowsPagefile < Chef::Resource
 
       provides(:windows_pagefile) { true }
 
-      description "Use the **windows_pagefile** resource to configure pagefile settings on Windows."
+      description "Use the **windows_pagefile** resource to configure pagefile settings on Windows. We are introducing a potentially breaking change - we no longer require or use a full path to a pagefile. We only need the drive (C:\, E:\, etc) you want to manage a pagefile for."
```

Comment:
putting the deprecation in the description is probably not right, that'll go out of date fairly quickly.

Reply:
Yep let's not do that

Reply:
corrected